src/Select/Select.stories.tsx

```diff
@@ -15,6 +15,8 @@ storiesOf('Select', module).add(
         invalid={boolean('invalid', false)}
         invalidText={text('invalidText', 'A valid value is required')}
         invalidTextColor={text('invalidTextColor', '')}
+        borderColor={text('borderColor', 'red')}
+        borderRadius={text('borderRadius', '4px')}
```

Comment:
Same here. Default to empty string so we see the default theme values when the story loads.